scripts/ansible/roles/linux/intel/vars/redhat/ootpa.yml

```diff
@@ -2,9 +2,9 @@
 # Licensed under the MIT License.
 
 ---
-intel_sgx_w_flc_driver_url: "https://download.01.org/intel-sgx/sgx-dcap/1.6/linux/distro/rhel8.0-Server/sgx_linux_x64_driver_1.33.bin"
-intel_sgx1_driver_url: "https://download.01.org/intel-sgx/sgx-linux/2.9.1/distro/rhel8.0-server/sgx_linux_x64_driver_2.6.0_95eaa6f.bin"
-intel_sgx_rpm_repo_tgz_url: "https://download.01.org/intel-sgx/sgx-linux/2.9.1/distro/rhel8.0-server/sgx_rpm_local_repo.tgz"
+intel_sgx_w_flc_driver_url: "https://download.01.org/intel-sgx/sgx-linux/2.10/distro/rhel8.1-server/sgx_linux_x64_driver_1.35.bin"
```

Comment:
We actually want the sgx-dcap drivers, the ones from sgx-linux do not have flc IIRC.

https://download.01.org/intel-sgx/sgx-dcap/1.9/linux/distro/rhel8.2-server/
https://download.01.org/intel-sgx/sgx-linux/2.12/distro/rhel8.2-server/

Are the URL's I believe.

Reply:
I was matching it up with the drivers we have for Ubuntu. If we want the latest, should I bump the Ubuntu drivers as well?

Reply:
Gotcha. I'm sure the binaries are the same, so no action required.

intel_sgx_w_flc_driver_url should be using the sgx-dcap driver though, Ubuntu is the [same](https://github.com/openenclave/openenclave/blob/4d56a217912e44bd5c0379f6306bbba4b553824f/scripts/ansible/roles/linux/intel/vars/ubuntu/xenial.yml#L5)